sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowWriter.java

```diff
@@ -59,7 +59,7 @@ public Result(ResourceId resourceId, long byteSize) {
     this.tempFilePrefix = basename;
   }
 
-  public final void open(String uId) throws Exception {
+  final void open(String uId) throws Exception {
     id = uId;
     resourceId = FileSystems.matchNewResource(tempFilePrefix + id, false);
     LOG.debug("Opening {}.", resourceId);
```

Comment:
noted! I think I'll leave optimizing the buffer sizes for another PR, as I would like to test the impact on performance first, but worth exploring.